modfiles/data/calculation/matrix_solver.lua

```diff
@@ -236,6 +279,12 @@ function matrix_solver.get_linear_dependence_data(subfactory_data, matrix_metada
                 allowed_free_items[item_key] = true
             end
         end
+
+        print("Fast free items", fast_allowed_free_items)
```

Comment:
Can you comment these out in the final version?

Reply:
Yes, everything below line 270 is old version + asserts for testing and should be removed